src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/types.js

```diff
@@ -1,7 +1,43 @@
 // @flow
-export type Item = {
-    title: string,
-    icon: string,
-    enabled?: boolean,
-    onClick?: () => void,
+
+export type BackButtonType = {
```

Comment:
I think I wouldn't add the `Type` suffix to everything... That's also not happening in the [flow documentation](https://flow.org/en/docs/types/aliases/). Same for the other types in here.